pkg/storage/storage.go

```diff
@@ -266,28 +275,56 @@ func (s *storageModule) initialize(policy pkg.StoragePolicy) error {
 		}
 	}
 
-	// make sure new pools are mounted
-	log.Info().Msgf("Making sure new volumes are mounted")
+	// make sure new pools are added to the list
 	for idx := range newPools {
-		if _, mounted := newPools[idx].Mounted(); !mounted {
-			log.Debug().Msgf("Mounting volume %s", newPools[idx].Name())
-			if _, err = newPools[idx].Mount(); err != nil {
-				s.brokenPools = append(s.brokenPools, pkg.BrokenPool{Label: newPools[idx].Name(), Err: err})
-				continue
-			}
-			s.volumes = append(s.volumes, newPools[idx])
-		}
+		s.pools = append(s.pools, newPools[idx])
 	}
 
 	if err := filesystem.Partprobe(ctx); err != nil {
 		return err
 	}
 
+	err = s.shutdownUnusedPools()
+	if err != nil {
+		return err
+	}
+
 	return s.ensureCache()
 }
 
+func (s *storageModule) shutdownDevice(devicePath string) error {
+	cmd := exec.Command("hdparm", "-y", devicePath)
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
+
+	return cmd.Start()
```

Comment:
Start does not wait for the command to finish execution. So your code will only print errors to stdout, but if the command fail to shutdown the device you will NOT receive an error back from this call. Instead you need to use cmd.Run() which will wait for the command to finish execution, and then return you an error if the command existed with a non 0 exit code.

```suggestion
	return cmd.Run()
```